### PR TITLE
Galar gyms and dungeons

### PR DESCRIPTION
**Pull request by Qwertypop04**
First pass at balancing Galar gyms and dungeons

### CODE REVIEW

File: src/modules/enums/Badges.ts [hunk 1]
```diff
@@ -110,18 +110,20 @@ enum BadgeEnums {
     'Galar_Water',
     'Galar_Fire',
     'Galar_Fighting',
+    'Galar_Ghost',
     'Galar_Fairy',
     'Galar_Rock',
+    'Galar_Ice',
```
Comment (RedSparr0w) [inline src/modules/enums/Badges.ts:116]: I'm not sure how the extra badges will work, or if this is the best way to add them yet.

Comment (Qwertypop04) [reply src/modules/enums/Badges.ts:116]: The badge case changes just made allow a region to have 10 badges without it looking weird. If we still want them to be a multiple of 4, we can always merge the badges later, Someone already made images for that.

Comment (RedSparr0w) [reply src/modules/enums/Badges.ts:116]: Sorry, wasn't meaning about having 10 badges, just having 2 different gym leaders from 2 different games.

Comment (Qwertypop4) [reply src/modules/enums/Badges.ts:116]: It is a little weird, but it's better than just randomly choosing one game, when everything else in pokeclicker either uses expanded versions or mixes both versions. And there's no expanded version for gen 8

**Comment by Jaaslet** [comment]
The code is failing, because 'Trainer Bede' is now called 'Gym Leader Bede' in the GymList, but not in the TownList (might have been lost in the merge. Wyndon Stadium got moved from PokemonLeague.ts to TownList.ts)

**Review by RedSparr0w — APPROVED** [review]
Looks good, thanks for the PR! 👍

**Pull request merged**